concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide columns.md

```diff
@@ -0,0 +1,103 @@
+TreeList can hide columns to adapt to screens of different sizes. To add this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) to **true**.
+
+Each column has a unique hiding priority. The rightmost column is the base column with the priority of 0. This value is incremented by 1 for columns from right to left; column with the lowest priority is hidden first.
+
+You can use the **columns[]**.[hidingPriority](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#hidingPriority) property to override the default behavior: specify custom hiding priorities for those columns only that you want to hide. In this case, the **columnHidingEnabled** property can be omited.
+
+[note]
+
+Peculiar cases to enable the **columnHidingEnabled** property:
+
+* TreeList truncates a column's cell text due to its length. This occurs when the column's [width](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#width) is not set or set as a percentage. If you need to hide columns until the full text is displayed, set the columnHidingEnabled property to **true**.
+
+* TreeList adds horizontal scrolling for adaptability when all columns have a fixed (columnWidth, columns[].width set in pixels) or adaptive (columnAutoWidth) widths. Set the columnHidingEnabled property to **true** if you need to disable the horizontal scrolling and hide columns instead.
```

Comment:
Why do all these details about widths matter? Can't we simply say:

"If this feature is enabled, the TreeList starts to hide columns when an alternative would be to truncate cell values or introduce horizontal scrolling."